music_assistant/server/providers/spotify/__init__.py

```diff
@@ -229,8 +229,35 @@ async def get_library_tracks(self) -> AsyncGenerator[Track, None]:
             if item and item["track"]["id"]:
                 yield await self._parse_track(item["track"])
 
+    async def _get_liked_songs_playlist(self) -> Playlist:
+        liked_songs = Playlist(
+            item_id="tracks",
+            provider=self.domain,
+            name="Liked Songs",  # TODO to be translated
+            owner="Me",  # TODO Get logged in user display name
+            provider_mappings={
+                ProviderMapping(
+                    item_id="tracks",
```

Comment:
please make this a constant (aso I think liked_songs is a better name)

Reply:
Makes me think: does it need to be globally unique? What if I link 2 different Spotify accounts?

Reply:
In that case append the instance id

Reply:
In that case append the instance id

Reply:
Done